packages/terra-cell-grid/src/terra-dev-site/test/cell-grid/Cell.test.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import {
+  Cell,
+} from '../../../index';
+
+const CellTest = () => (
+  <React.Fragment>
```

Comment:
We should render this as a div instead of a fragment. By rendering a fragement, only the first cell is getting captured in the wdio screenshot